### ISSUE TITLE
Some buttons are missing or have ambiguous accessibility labels

### ISSUE THREAD

**Issue by ecnepsnai** [certificate-helper/TLS-Inspector#215]
Some buttons, such as the "options" menu item, the "close" menu item, the "inspect" menu item are missing accessibility labels or are ambiguous to users with voiceover enabled.
